lib/implore.es6.js

```diff
@@ -121,7 +121,16 @@ export default function implore(request, callback) {
 			response.body = xhr.responseText;
 			response.status = xhr.status;
 			response.type = xhr.getResponseHeader('Content-Type');
-			response.headers = xhr.getAllResponseHeaders();
+			const rawHeaders = xhr.getAllResponseHeaders();
+			// Just in case some headers don't get processed properly
+			response.rawHeaders = rawHeaders;
+			// Processed headers as a hash for easier use
+			const responseHeaders = rawHeaders.split("\n");
+			response.headers = {};
+			for (let header of responseHeaders) {
+				const parts = header.split(': ');
+				response.headers[parts[0]] = parts[1];
+			}
```

Comment:
As a more general comment, why not just make this a bit cleaner/easier to read by setting `response.rawHeaders` to the result of `xhr.getAllResponseHeaders()` (as you had it originally) and then simply setting `response.headers` to the result of running `reduce` on the result of`response.rawHeaders.split('\n')`? That way, this whole block of code essentially becomes two assignments to properties of the `response` object (just like the LOC immediately before it). I know it results in one additional lookup to get the value of `response.rawHeaders` (when you do the `split` before the `reduce`), but that's an extremely minor thing.